releasenotes/notes/otel-accesslog-formatter.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: telemetry
+releaseNotes:
+  - |
+    **Added** access log formatter support for OpenTelemetry.
+upgradeNotes:
```

Comment:
This is not an upgrade note. An upgrade note means "when you upgrade, something is going to break".

This is saying "when you upgrade, you can now use a feature" which should be a release note